src/oscar/apps/offer/utils.py

```diff
@@ -33,3 +38,20 @@ def load_proxy(proxy_class):
     except AttributeError:
         raise exceptions.ImproperlyConfigured(
             "Module %s does not define a %s" % (module, classname))
+
+
+@lru_cache(maxsize=None)
+def get_offer_rounding_function():
```

Comment:
I think it would be worth making this a core util usable for loading any dotted path string with caching?

I'd suggest dropping the deprecation warning, and make this a 2.0 breaking change?